test/integration/build/aws.tf

```diff
@@ -919,6 +919,11 @@ resource "aws_db_instance" "db_rds" {
   }
 }
 
+resource "aws_db_snapshot" "test" {
```

Comment:
could we have more unique resource alias name instead of `test` to avoid future conflicts

Reply:
fixed